doc/architecture/timestamps_overview.adoc

```diff
@@ -0,0 +1,67 @@
+ifndef::include-only-once[]
+:root-path: ../
+include::{root-path}_config.adoc[]
+endif::[]
+= Timestamps
+
+== Simulation time
+
+`Simulation time` refers to the timestamp within a simulation environment.
+
+**Definition of zero time of simulation time**
+- Zero time is arbitrary but must be identical for all frames and corresponding messages (probably within one simulation).
+- Zero time does not need to coincide with the unix epoch.
+- It is recommended to use the start time of the simulation as zero time.
+
+**SensorData-specific timestamp**
+- Point in time that the sensor data message becomes available to the rest of the system (i.e. the driving functions).
+- Corresponds with the sending time / publish time.
+- Takes into account internal sensor processing latency.
+- Does not include bus communication delays (e.g. latency) that occur after the sensor output.
+
+**HostVehicleData-specific timestamp**
+- Time at which the host vehicle data message becomes available as a snapshot of the on-board network information.
+
+**Other top-level message timestamps**
+- GroundTruth, SensorView, TrafficCommand, TrafficUpdate, TrafficCommandUpdate, StreamingUpdate
+- The Timestamp coincides both with the notional simulation time the data applies to and with the time it was sent / published.
+- No inherent latency for the corresponding data, as opposed to e.g. SensorData-specific timestamp.
+
+== Vehicle-global synchronized time::
```

Comment:
This must/should be using the same same time reference as Simulation time` right? If yes, should we make that more clear?

Reply:
I'm not sure if that is the case. I think there could be use cases where you want to have an independent vehicle time course (e.g. simulation of timing issues/failures). As long as the simulation environment can handle the differences, this would be fine IMO. If you don't want to simulate that, you can of course use the same time frame.

Reply:
But even for timing issues/failures this is still the same time reference/zero time? I am not saying it has to be the same stamp (which allows you to consider jitter etc.).

Reply:
Sorry, I think I made it not clear or do not fully understand the standard. I do not mean that this must be the same value as the step value of the current simulated time. I can be a different value to all kinds of reasons you mentioned but it must refer to the same zero time?

On this page here `SensorData-specific timestamp` is mentioned below the `Simulation time`. This includes internal latency etc. Then why is e.g. `DetectedEntityHeader/measurement_time` something that is vehicle-global synchronized - or why is  `SensorData-specific timestamp` not? Why the split in the documentation?

The current class documentation of `DetectedEntityHeader/measurement_time` even says 

> ... in the global synchronized time.

an not 

> ... in the vehicle-global synchronized time.

Reply:
> Sorry, I think I made it not clear or do not fully understand the standard. I do not mean that this must be the same value as the step value of the current simulated time. I can be a different value to all kinds of reasons you mentioned but it must refer to the same zero time?

It cannot refer to the same zero time, as that relationship can change over time: Simulated time proceeds in uniform fashion, whereas whatever the vehicle thinks its vehicle-global synchronized time is can change all of the time - consider resync with GPS clock after some frequency drift without GPS clock connection.

Or in other words: Simulation time (based on zero time), simulated real-time, etc. are under the simulations control and proceed with full regularity (ignoring leap seconds for now). Vehicle-glocal synchronized time is whatever the vehicle systems decide it is - it has nothing to do with the simulation, but all with the vehicle software and its interactions with simulated reality.

> On this page here `SensorData-specific timestamp` is mentioned below the `Simulation time`. This includes internal latency etc. Then why is e.g. `DetectedEntityHeader/measurement_time` something that is vehicle-global synchronized - or why is `SensorData-specific timestamp` not? Why the split in the documentation?

Because SensorData.timestamp is for use by the simulation to ensure simulated timing is correct (it is not something that exists in the real world), whereas DetectedEntityHeader is purely for the vehicle system consumption (it is payload that exists in the real world), we do not care what it is from the point of view of the simulation.

They are two very different things.

> The current class documentation of `DetectedEntityHeader/measurement_time` even says
> 
> > ... in the global synchronized time.
> 
> an not
> 
> > ... in the vehicle-global synchronized time.

Probably an editing oversight, as global synchronized time is supposed to be the vehicle's global synchronized time (might have happened during integration of the ISO 23150 wording).